### PR TITLE
A note about user restrictions of viewing posted documents with RC was changed

### PR DESCRIPTION
**Pull request by AleksanderGladkov**
In 2010 it was prohibited for a user to view any posted document that is not related to user's responsibility center. This restriction is still actual, so the note must be changed to match the current behaviour.
However, it is still possible for a user to view all Customer/Vendor Ledger Entries.

I checked the behaviour on Business Central Spring Release and Business Central Wave 2 2020.

Feel free to suggest any changes if my change does not correspond with the documentation style etc.

### CODE REVIEW

**Comment by AleksanderGladkov** [comment]
@edupont04 I don't understand why Greetings script fails on this pull request. Maybe you know what should I do to avoid this failure? Thanks.

**Comment by edupont04** [comment]
@AleksanderGladkov it's a technical issue on the backend, and has no impact on your PR. We have added the Greetings action as a service to new contributors, but it's been failing repeatedly across many repos in the past week or so.

I'm waiting for confirmation from the product team before I can merge your PR. But thank you for your contribution!

**Review by edupont04 — APPROVED** [review]
Confirmed by engineering.

**Pull request merged**
